components/remote_retrieval/actions/get-pending-order/get-pending-order.mjs

```diff
@@ -0,0 +1,29 @@
+import app from "../../remote_retriever.app.mjs";
+
+export default {
+  key: "remoteretriever-get-pending-order",
+  name: "Get Pending Order",
+  description: "These are the orders for which the payment process has not been completed.[See the documentation](https://www.remoteretrieval.com/api-documentation/#pending-orders)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    app,
+  },
+  methods: {
+    getPendingOrder() {
+      return this.app.makeRequest({
+        path: "/pending-orders/",
+      });
+    },
+  },
```

Comment:
The method `pendingOrders` using the endpoint `/pending-orders/` already exists within the app file (remote_retrieval.app.mjs).
```suggestion
```